src/oss/confluent.sh

```diff
@@ -686,6 +755,43 @@ config_service() {
     fi
 }
 
+enable_metrics_reporter() {
```

Comment:
Also, is this done for COS?  From what I can tell, it is.  But I think we may want it only for CE.

Reply:
It is not. In both functions added, there's a call to `is_enterprise` at the beginning to enable this appropriately.

Reply:
I see that now, I'm sorry I missed it.  It is very obvious now ;)